app/controllers/cloud_volume_controller.rb

```diff
@@ -602,7 +602,7 @@ def cinder_manager_options
     options[:volume_type] = params[:volume_type] if params[:volume_type]
     cloud_tenant = find_record_with_rbac(CloudTenant, cloud_tenant_id)
     options[:cloud_tenant] = cloud_tenant
-    options[:ems] = cloud_tenant.ext_management_system
+    options[:ems] = cloud_tenant.ext_management_system.cinder_manager
```

Comment:
Note that this breaks the functionality we're removing in a future PR :exclamation:

Reply:
Which functionality is that?

Reply:
Oh right, it's not :laughing: sorry